plugins/woocommerce-blocks/assets/js/base/components/cart-checkout/shipping-calculator/context.ts

```diff
@@ -7,7 +7,9 @@ type ShippingCalculatorContextType = {
 	shippingCalculatorID: string;
 	showCalculator: boolean;
 	isShippingCalculatorOpen: boolean;
-	setIsShippingCalculatorOpen: ( value: boolean ) => void;
+	setIsShippingCalculatorOpen: React.Dispatch<
```

Comment:
Small tweak to this type to better match the type used for set state actions elsewhere